jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/JibPlugin.java

```diff
@@ -167,17 +166,17 @@ public void apply(Project project) {
             TaskProvider<Task> warTask = TaskCommon.getWarTaskProvider(project);
             TaskProvider<?> dependsOnTask;
             if (warTask != null) {
-              TaskProvider<ExplodedWarTask> explodedWarTask =
+              /*TaskProvider<ExplodedWarTask> explodedWarTask =
                   tasks.register(EXPLODED_WAR_TASK_NAME, ExplodedWarTask.class);
               explodedWarTask.configure(
                   task -> {
                     task.dependsOn(warTask);
                     task.setWarFile(((War) warTask.get()).getArchivePath().toPath());
                     task.setExplodedWarDirectory(
                         GradleProjectProperties.getExplodedWarDirectory(projectAfterEvaluation));
-                  });
+                  });*/
```

Comment:
We can remove the commented code.

Reply:
Removed it. Also marked ExplodedWarTask as deprecated for the time being.